src/util/execCommand.js

```diff
@@ -13,6 +12,8 @@ const isWindows = os.platform() === 'win32';
 function execCommand(exec, command, ...args) {
 	exec = exec || spawnSync;
 	command = isWindows ? `${command}.cmd` : command;
+	args[0] = isWindows ? escapeArguments(args[0]) : args[0];
```

Comment:
Now it's a function, you need to call it.

Reply:
Instead of exporting a function, should we export the boolean directly?

```js
module.exports = os.platform() === 'win32';
```

Reply:
I don't have a preference, both is fine I think ;-)